splunk/scripts/log_indexing.sh

```diff
@@ -7,7 +7,10 @@ SPLUNK_PASSWORD="Password"
 echo "Indexing logs..."
 for file in "$logs_path"/*; do
   echo "Indexing $file"
-  curl  --insecure --user admin:"$SPLUNK_PASSWORD" "$endpoint" --data name="$file" --data index="test_index"
+  curl --insecure --user admin:"$SPLUNK_PASSWORD" "$endpoint" --data name="$file" --data index="test_index" || {
+    echo "Encountered an error while initiating indexing for $file. Exiting."
+    exit 1
```

Comment:
Wouldn't it be more reliable to retry rather then just exit?

Reply:
Yes, it is possible. I'll add it